safe/utilities/utilities.py

```diff
@@ -49,7 +49,6 @@
 def get_error_message(exception, context=None, suggestion=None):
```

Comment:
Yes, @timlinux I think you are talking of that one. But this one is doing too much as I don't want the the Messaging system.